gen/nvml/device.go

```diff
@@ -953,18 +979,44 @@ func (Device Device) GetComputeRunningProcesses() ([]ProcessInfo, Return) {
 
 // nvml.DeviceGetGraphicsRunningProcesses()
 func DeviceGetGraphicsRunningProcesses(Device Device) ([]ProcessInfo, Return) {
+	var Infos []ProcessInfo  // This is the v2 version of process info data structure
 	var InfoCount uint32 = 1 // Will be reduced upon returning
+	var ret = SUCCESS        // Will be changed upon returning
 	for {
-		Infos := make([]ProcessInfo, InfoCount)
-		ret := nvmlDeviceGetGraphicsRunningProcesses(Device, &InfoCount, &Infos[0])
+		if usesNvmlDeviceGetComputeRunningProcesses_v1 {
```

Comment:
This should be `usesNvmlDeviceGetGraphicsRunningProcesses_v1`.

Reply:
Thanks. I'll fix this right away.